spec/integration/configuring_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe "Requiring govuk_app_config" do
   it "correctly initialises error tracking" do
-    ClimateControl.modify SENTRY_CURRENT_ENV: "integration-or-somesuch" do
+    ClimateControl.modify SENTRY_CURRENT_ENV: "integration-or-somesuch", GOVUK_DATA_SYNC_PERIOD: "22:00-08:00" do
```

Comment:
Is this change necessary?

Reply:
Looks like it's not needed anymore (it was at one point). Thanks!